src/pages/Error.tsx

```
@@ -1,56 +1,42 @@
-          style={{
-            color:
-              document.body.style.backgroundColor === "rgb(255, 255, 255)"
-                ? "#000"
-                : "#fff",
-          }}
-        />
-      </a>
-    </ErrorParagraph>
-  </ErrorContainer>
-);
+function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)